src/Concordium/Client/Output.hs

```diff
@@ -465,6 +484,23 @@ printModuleInspectInfo CI.ModuleInspectInfo{..} = do
             showReceives [] = []
             showReceives ((fname, mSchema):remaining) = indentBy 4 (showContractFuncV2 fname mSchema) : showReceives remaining
 
+    showContractSigsV3 :: Map.Map Text CI.ContractSigsV3 -> [String]
+    showContractSigsV3 = go . sortOn fst . Map.toList
+      where go [] = []
+            go ((cname, CI.ContractSigsV3{..}):remaining) = [showContractFuncV2 cname csv3InitSig]
+                                                          ++ showReceives (sortOn fst . Map.toList $ csv3ReceiveSigs)
+                                                          ++ showEvents cs3EventSchema
+                                                          ++ go remaining
+
+            showReceives :: [(Text, Maybe CS.FunctionSchemaV2)] -> [String]
+            showReceives [] = []
+            showReceives ((fname, mSchema):remaining) = indentBy 4 (showContractFuncV2 fname mSchema) : showReceives remaining
```

Comment:
Good point! I actually wanted to do something like that, but since it was not already done, I adhered to following the style of the code. Is such refactoring encouraged, however?

Reply:
The v2 code was written in quite a rush (by me), so that is why it wasn't done. Or perhaps I simply overlooked it ;) 
Well, that depends. In this simple and isolated case, I think it's fine, but refactoring should provide a decent amount of benefits before it is worth it. Because it is not just the refactoring itself that takes time. There is also the time needed for reviewing the changes, retesting the changes, and time needed for other developers to learn the new API or semantics.
So if in doubt, ask ;)

Reply:
Makes sense :-) I agree refactoring makes sense here, and I'll add it here, and in the future I'll ask if I'm in doubt 👍